tubular/tests/test_edx_api.py

```diff
@@ -96,15 +318,244 @@ def test_retirement_partner_cleanup_backoff_on_connection_error(
             mock_retirement_partner_cleanup
     ):
         mock_backoff_handler.side_effect = BackoffTriedException
+        response = requests.Response()
+        response.status_code = svr_status_code
+        mock_retirement_partner_cleanup.retirement_partner_cleanup.side_effect = ConnectionError(
+            response=response)
```

Comment:
please put trailing parenthesis down to the new line

Reply:
Done